src/main/scala/zio/prelude/Bicovariant.scala

```diff
@@ -0,0 +1,56 @@
+package zio.prelude
+
+trait RightMap[:=>[_,+_]] {
```

Comment:
Wouldn't a better name be `RightCovariant`? To be consistent with the other names...

Reply:
Nice name 👍  thanks.
Renamed: https://github.com/zio/zio-prelude/pull/279/commits/bb2059936e9f205d571a760ab846c020ad230eb9